packages/core/src/physics/DynamicCollider.ts

```diff
@@ -46,11 +237,34 @@ export class DynamicCollider extends Collider {
     (<IDynamicCollider>this._nativeCollider).addTorque(torque);
   }
 
+  /**
+   * Moves kinematically controlled dynamic actors through the game world.
+   * @param position The desired position for the kinematic actor
+   * @param rotation The desired rotation for the kinematic actor
+   */
+  setKinematicTarget(position: Vector3, rotation: Quaternion): void {
```

Comment:
Use `move` insteadof `setKinematicTarget ` and add function overload:

-  `move(position: Vector3, rotation: Quaternion)`
-  `move(position: Vector3)`
-  `move(rotation: Quaternion)`

Reply:
+1